bindings/python-sdk/python_sdk.py

```diff
@@ -0,0 +1,341 @@
+import io
+import time
+from typing import Any, Dict, List, Optional, Set, Tuple
+
+from pathlib import Path
+from pprint import pprint
+from determined.common import api, constants, context, yaml
+
+import swagger_client
+from swagger_client.api.authentication_api import AuthenticationApi  # noqa: E501
+from swagger_client.rest import ApiException
+from swagger_client import models
+from swagger_client.models import V1CreateExperimentRequest as CreateExperimentRequest
+from swagger_client.models import V1CreateExperimentResponse as CreateExperimentResponse
+from swagger_client.models import V1File as V1File
+from swagger_client.models import V1GetExperimentResponse as GetExperimentResponse
+from swagger_client.models import V1GetExperimentsResponse as GetExperimentsResponse
+from swagger_client.models import V1GetExperimentTrialsResponse as GetExperimentTrialsResponse
+from swagger_client.models import V1GetTrialResponse as GetTrialResponse
+from swagger_client.models import V1GetExperimentLabelsResponse as GetExperimentLabelsResponse
+from swagger_client.models import Determinedexperimentv1State as ExperimentState
+
+def _print_exception(e: ApiException):
+    print("{} {} {}".format(e.status, e.reason, e.body))
+
+def _parse_config_file(config_file: io.FileIO) -> Dict:
```

Comment:
What happens if the config file doesn't load for some reason? We probably need some more error-handling here.

Reply:
will add error handling as needed.